test/unit/attributes.js

```diff
@@ -1117,7 +1139,7 @@ QUnit.test( "val(select) after form.reset() (Bug #2551)", function( assert ) {
 } );
 
 QUnit.test( "select.val(space characters) (gh-2978)", function( assert ) {
-	assert.expect( 35 );
+	assert.expect( 37 );
```

Comment:
Yeah, sure, i'm saying it would be better not to change the tests, but add different ones. If we would use the terminology, previously that was an unit-test, now we found a bug, so it should be a regression test, so it might make sense not to unite those